example/code/glob_avg.py

```diff
@@ -0,0 +1,55 @@
+'''
+Creates a monthly temparature time series for a global average
+temperature, using of all Berkeley region data per time index. The
+average is of the temperature absolute value, the anomaly plus the
+base value for a time/region.  Missing data is interpolated using
+Pandas interpolate(). Data is from http://berkeleyearth.org/data/, see
+"Monthly Land" under the Gridded section, and use the "Equal Area"
+data file.
+
+'''
+import pandas as pd, gc, netCDF4, psutil 
+import csv, numpy as np
+import matplotlib.pyplot as plt
+
+url='Complete_TAVG_EqualArea.nc'
+nc = netCDF4.Dataset(url)
+clim = nc['climatology'][:,:]
+anom =  nc['temperature'][:,:]
+time =  nc['time'][:]
+
+def year(float_year):
+    year = int(float_year)
+    return year
+
+def month(float_year):
+    month = int((float_year % 1) * 12)
+    return month
+
+monidxs = np.array(list(map(month, time)))
+
+tmp_clims = np.zeros(time.shape[0])
+time_idxs = np.array(range(time.shape[0]))
+res = []
+for region in range(anom.shape[1]):
+    base_temps = clim[monidxs, region]
+    real_temps = base_temps + anom[:, region]
+    tmp = pd.Series(real_temps)
+    tmp = tmp.replace('--', np.nan)
+    tmp = tmp.interpolate(method='linear',limit_direction='backward')
+    res.append(list(tmp))
+
+arr = np.array(res)
+df = pd.DataFrame(res)
+
+avg = pd.DataFrame(df.median(axis=0))
```

Comment:
Add comments about the structure of the data in `df` and `avg`.

Reply:
I'll let the next user print out the contents for that one :)